rpgJavaInterpreter-core/build.gradle

```diff
@@ -248,4 +252,44 @@ for further informations."""
     }
 }
 
-check.dependsOn runMutes
+task profileMainClass (type: JavaExec) {
```

Comment:
I think there is a small mistake here: the check.dependsOn runMutes should not be deleted

Reply:
You are right.